### PR TITLE
[#2036] Fix dimension checks

### PR DESCRIPTION
**Pull request by cadecairos**
https://webmademovies.lighthouseapp.com/projects/65733/tickets/2036-image-and-wikipedia-topleft-checks-fail-on-0

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @cadecairos received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/5f06c602fc7834b/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @cadecairos received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/2e98039ca554e9e/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/5f06c602fc7834b/output.txt

Total script time: 1.03 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/2e98039ca554e9e/output.txt

Total script time: 0.74 mins

Server is running on http://pr816.butyr.org/templates/basic/
Run the Test Suite: http://pr816.butyr.org/test/

File: templates/assets/plugins/wikipedia/popcorn.wikipedia.js [hunk 1]
```diff
@@ -30,6 +30,14 @@
     return fragment;
   }
 
+  function validateDimension( value, fallback ) {
+    if ( typeof value === "number" ) {
+      return value;
+    }
+    return fallback;
+  }
```
Comment (ScottDowne) [inline templates/assets/plugins/wikipedia/popcorn.wikipedia.js:34]: We should put this function in a shared area.

File: templates/assets/plugins/wikipedia/popcorn.wikipedia.js [hunk 2]
```diff
@@ -30,6 +30,14 @@
...
+
+
```
Comment (ScottDowne) [inline templates/assets/plugins/wikipedia/popcorn.wikipedia.js:36]: Why the extra whitespace?

File: templates/assets/plugins/wikipedia/popcorn.wikipedia.js [hunk 3]
```diff
@@ -55,10 +63,11 @@
       options._container = _container = create( "div" );
 
       _container.classList.add( "wikipedia-inner-container" );
-      _container.style.width = normalize( options.width || _manifestOpts.width[ "default" ], 20, 100 ) + "%";
-      _container.style.height = normalize( options.height || _manifestOpts.height[ "default" ], 30, 100 ) + "%";
-      _container.style.top = normalize( options.top || _manifestOpts.top[ "default" ], 0, 70 ) + "%";
-      _container.style.left = normalize( options.left || _manifestOpts.left[ "default" ], 0, 80 ) + "%";
+
```
Comment (ScottDowne) [inline templates/assets/plugins/wikipedia/popcorn.wikipedia.js:62]: is normalize a dead function now?

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @ScottDowne received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/aa5efc653aab4e4/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @ScottDowne received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/4abceffb76b6d2a/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/aa5efc653aab4e4/output.txt

Total script time: 0.52 mins

Server is running on http://pr816.butyr.org/templates/basic/
Run the Test Suite: http://pr816.butyr.org/test/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/4abceffb76b6d2a/output.txt

Total script time: 0.88 mins

**Pull request merged**